### PR TITLE
Fix deneb doppelganger tests

### PR DESCRIPTION
**Pull request by pawanjay176**
## Issue Addressed

N/A

## Proposed Changes

Fixes doppelganger scripts. ~~Also makes the code compile by adding an `unimplemented` block that should be resolved in #4092~~

### CODE REVIEW

**Comment by pawanjay176** [comment]
Hoping this last run should pass CI :crossed_fingers:

File: .github/workflows/local-testnet.yml [hunk 1]
```diff
@@ -26,7 +26,16 @@ jobs:
             repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Install ganache
         run: npm install ganache@latest --global
-
+      - name: Install geth
```
Comment (amakrushin) [inline .github/workflows/local-testnet.yml:29]: maybe good to mention in setup.md https://github.com/sigp/lighthouse/blob/unstable/book/src/setup.md?plain=1#L12

Comment (pawanjay176) [reply .github/workflows/local-testnet.yml:29]: Hey @atokama. We are using this for testing for 4844 specific stuff. This might not get merged for quite some time to the main branch. We have another PR https://github.com/sigp/lighthouse/pull/3807 where we should be ideally adding this and updating the docs for building post merge local testnets. Haven't been able to get around to that yet though :sweat_smile:

**Comment by pawanjay176** [comment]
Update: the failure scenario is not working for some reason despite logs indicating it should. Still checking it out

**Comment by divagant-martian** [comment]
An update from inner talks. This is failing due to an expected/correct peer scoring event (peer is being banned). Work on https://github.com/sigp/lighthouse/pull/4135 should unblock this

File: scripts/tests/vars.env [hunk 2]
```diff
@@ -48,13 +56,7 @@ SECONDS_PER_SLOT=3
 SECONDS_PER_ETH1_BLOCK=1
 
 # Proposer score boost percentage
-PROPOSER_SCORE_BOOST=40
+PROPOSER_SCORE_BOOST=70
```
Comment (realbigsean) [inline scripts/tests/vars.env]: Why this change? it's 40 in the spec: https://github.com/ethereum/consensus-specs/blob/dev/configs/mainnet.yaml#L88
